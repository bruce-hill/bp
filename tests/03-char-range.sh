# Character sets and ranges work with backticks
# Example: bp -p '`a-z,A-Z' matches all lowercase and uppercase letters
bp -p '`0-9,a-f'
