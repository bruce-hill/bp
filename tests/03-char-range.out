0
1
2
3
a
f
